pkg/certificate/mrc_reconciler.go

```diff
@@ -80,6 +63,28 @@ func getSigningAndValidatingMRCs(mrcList []*v1alpha2.MeshRootCertificate) (*v1al
 	return mrcList[0], mrcList[1], nil
 }
 
+// ValidateMRCCombination takes a list of MRCs and ensures that the MRC combination is valid
```

Comment:
Can we add a note that this method expects the list of MRCs to only include MRCs with active or passive intent?